mikeio1d/result_network/result_catchments.py

```diff
@@ -80,7 +96,18 @@ def to_geopandas(self) -> GeoDataFrame:
 
         ids = self._catchment_ids
         geometries = self._geometries
-        data = {"id": ids, "geometry": geometries}
+        data = {"name": ids, "geometry": geometries}
```

Comment:
id vs. name: why do you think name is preferred?

Reply:
Ok, I see. I think it makes more sense to use name, because id is a built-in function name.

Reply:
> Ok, I see. I think it makes more sense to use name, because id is a built-in function name.

`id` is a built-in function which is bad idea to override (Python is a bit too flexible in my taste...). But I don't see any problem with using `"id"` as key name in a dict. That must be very common when dealing with database records.